src/api/firebase.js

```diff
@@ -89,15 +91,35 @@ export async function addItem(listId, { itemName, daysUntilNextPurchase }) {
 	}
 }
 
-export async function updateItem(listId, id, totalPurchases) {
+export async function updateItem(listId, item) {
+	let daysSinceLastPurchase;
+	if (item.totalPurchases < 2) {
+		daysSinceLastPurchase = item.estimatedDaysUntilNextPurchase;
+	} else {
+		daysSinceLastPurchase = getDaysBetweenDates(
+			new Date(item.dateLastPurchased?.seconds * 1000), // Convert from seconds to milliseconds and then to a Date
```

Comment:
There's a firebase method, [.toDate()](https://firebase.google.com/docs/reference/js/v8/firebase.firestore.Timestamp#todate), that converts a firebase timestamp into a Date object.  You could replace `.seconds * 1000` with `.toDate()` if you wanted to here.